docs/csharp/whats-new/csharp-7.md

```diff
@@ -198,10 +198,9 @@ enables inspecting an object and its properties to determine if that object
 satisfies the sought pattern. You use the `when` keyword to specify additional
 rules to the pattern.
 
-### `is` expressions
+### `is` expression
 
-The `is` pattern expressions extend the familiar `is` operator
-to query a object beyond its type.
+The `is` expression extends the familiar `is` operator to query an object beyond its type.
```

Comment:
I think "pattern expression" is fine -- an expression that conforms to a particular pattern. "Pattern expression" is also the term we've used to describe the supported patterns in the [Pattern Expressions](https://docs.microsoft.com/en-us/dotnet/articles/csharp/pattern-matching) topic.